tokio/src/io/async_fd.rs

```diff
@@ -835,6 +839,8 @@ impl<'a, Inner: AsRawFd> AsyncFdReadyGuard<'a, Inner> {
     /// Indicates to tokio that the file descriptor no longer has a specific readiness.
     /// The internal readiness flag will be cleared, and tokio will wait for the
     /// next edge-triggered readiness notification from the OS.
+    /// This implies that if a readiness notification occurs following the last operation
+    /// but prior to invoking `clear_ready`, it will not be cleared.
     ///
```

Comment:
This part isn't on all of them? But I think the other change is probably sufficient.

Reply:
Ah yes it was a leftover comment. Thanks.